client/.eslintignore

```diff
@@ -0,0 +1 @@
+webpack/*
```

Comment:
should also include the build dir here since we won't be linting the output files.

Reply:
Added build